controllers/configurationpolicy_controller_test.go

```diff
@@ -627,6 +640,16 @@ func TestShouldEvaluatePolicy(t *testing.T) {
 			policyv1.EvaluationInterval{NonCompliant: "12h"},
 			policyv1.NonCompliant,
 			false,
+			nil,
+		},
+		{
+			"Deletion timestamp is non nil",
+			time.Now().UTC().Add(-12 * time.Second).Format(time.RFC3339),
```

Comment:
This will make it so that we know it's not based on the evaluation interval not being reached.
```suggestion
			time.Now().UTC().Add(-13 * time.Hour).Format(time.RFC3339),
```